apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c

```diff
@@ -628,6 +628,11 @@ static void utc_audio_pcm_readi_p(void)
 
 	while (remain > 0) {
 		frames_read = pcm_readi(g_pcm, buffer, pcm_get_buffer_size(g_pcm));
+		if(frames_read == -EPIPE){
+			pcm_prepare(g_pcm);
```

Comment:
why don't you add pcm_prepare for writei function?
I think continue is ok for for readi, but writei is diffrent.
You can adjust do-while & TC_ASSERT sentence, I think